lib/collectionHolder.ts

```diff
@@ -109,20 +109,18 @@ export const getAvailableFreeMints = async (
   }
 }
 
-export const mintByCollectionHolder = async (signer: Signer, passportIds: any) => {
+export const mintByCollectionHolder = async (signer: Signer, passportIds: any, to: string) => {
   const contract = new ethers.Contract(
     process.env.NEXT_PUBLIC_COLLECTION_HOLDER,
     collectionHolderAbi,
     signer,
   )
   try {
-    const address = await signer.getAddress()
     const tx = await contract.mint(
-      passportIds,
+      [passportIds[passportIds.length - 1]],
```

Comment:
what are you trying to do here?

Reply:
fixed with your tip to use `availablePassportIds` ✅ 

<img width="873" alt="Screenshot 2023-08-01 at 9 08 43 PM" src="https://github.com/defientco/CRE8ORS-UI/assets/23249402/68a1842c-9cca-4451-82ca-79866e5c0422">